app/javascript/packs/menus.js

```diff
@@ -0,0 +1,15 @@
+$(document).on('turbolinks:load', function() {
```

Comment:
please, try to avoid jQuery. Plain JS gives us everything we need nowadays.
also, would be supercool if you made this code into JS class. 

```
class AccountsTable {
  constructor() {
    this.switchers = document.querySelectorAll('.currency-switch');
    this.tabsSwitchers = document.querySelectorAll('.currency-tab-switch');
  }

  init() {
    self = this;

    this.switchers.forEach(switcher => {
      switcher.addEventListener('click', function() {
        let index = this.getAttribute('data-index');
        let parent = this.closest('.currency-parent-container');

        self.toggleVisibility(parent, index, 'current', 'total', 'index');
        self.toggleVisibility(parent, index, 'current', 'switch', 'index');
        self.toggleVisibility(parent, index, 'current', 'accounts-block', 'index');
      });
    });
....

==========

  new Creditor.AccountsTable().init();
```

look at this beauty <3